homeassistant/components/bunq/sensor.py

```diff
@@ -0,0 +1,146 @@
+"""Support for bunq account balance."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from bunq.sdk.context import ApiContext, ApiEnvironmentType, BunqContext
+from bunq.sdk.exception import ApiException, BunqException
+from bunq.sdk.model.generated import endpoint
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_API_KEY
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+
+ICON = "mdi:cash-multiple"
+UPDATE_INTERVAL = timedelta(seconds=60)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_API_KEY): cv.string})
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up bunq sensors."""
+    sensors = []
+
+    try:
+        # setup api
+        api_context = ApiContext(
+            ApiEnvironmentType.PRODUCTION, config.get(CONF_API_KEY), "Home Assistant",
```

Comment:
Use dict[key] for required config keys:
```suggestion
            ApiEnvironmentType.PRODUCTION, config[CONF_API_KEY], "Home Assistant",
```